src/spaceObjects/playerSpaceship.cpp

```diff
@@ -1188,11 +1188,11 @@ void PlayerSpaceship::closeComms()
                 if (player_ship->comms_state == CS_BeingHailed && player_ship->comms_target == this)
                 {
                     player_ship->comms_state = CS_Inactive;
-                    player_ship->addToShipLog("Hailing from " + getCallSign() + " stopped", colorConfig.log_generic);
+                    player_ship->addToShipLog(tr("shiplog", "Hailing from ") + getCallSign() + tr("shiplog", " stopped"), colorConfig.log_generic);
```

Comment:
Another that needs `format`, this should make it more clear why it is important to use `format`